backend/src/flotilla/database/crud.py

```diff
@@ -47,8 +54,10 @@ def read_report_by_id(db: Session, report_id: int) -> ReportDBModel:
     return report
 
 
-def read_events(db: Session) -> List[EventDBModel]:
-    events: List[EventDBModel] = db.query(EventDBModel).all()
+def read_events(
+    db: Session, skip: int = settings.SKIP, limit: int = settings.LIMIT
+) -> List[EventDBModel]:
+    events: List[EventDBModel] = db.query(EventDBModel).offset(skip).limit(limit).all()
```

Comment:
Consider changing the query parameter names to the more intuitive "page" & "pageSize"/"count"

Reply:
The input should also be validated, for example the skip/page and lmit/count should both be integers >= 0, and as mentioned in another commend the count/limit <= MAX_LIMIT